Makefile

```diff
@@ -82,7 +82,7 @@ APPNAME ="Zcash"
 APP_LOAD_PARAMS += --path $(APP_PATH)
 else ifeq ($(COIN),horizen)
 # Horizen
-DEFINES   += BIP44_COIN_TYPE=121 BIP44_COIN_TYPE_2=121 COIN_P2PKH_VERSION=8329 COIN_P2SH_VERSION=8342 COIN_FAMILY=4 COIN_COINID=\"Horizen\" COIN_COINID_HEADER=\"HORIZEN\" COIN_COINID_NAME=\"Horizen\" COINID=$(COIN) COIN_COINID_SHORT=\"ZEN\" COIN_KIND=COIN_KIND_HORIZEN
```

Comment:
Hi ! 
I don't think this is the way to solve it. 
Maybe what you want to do is to have a conditional definition of the `SIGNMAGIC` array value defined [here](https://github.com/LedgerHQ/app-bitcoin/blob/61f525d32e07be05ff696c9e908aecc4b5879b76/src/btchip_rom_variables.c#L43) ? 
Something like 
```c
#if (COIN_COINID == "Horizen")
unsigned char const SIGNMAGIC[] = {' ', 'Z', 'c', 'a', 's', 'h', ' ', 'S', 'i', 'g', 'n', 'e', 'd', ' ', 'M',
                                   'e', 's', 's', 'a', 'g', 'e', ':', '\n'};
#else
unsigned char const SIGNMAGIC[] = {' ', 'S', 'i', 'g', 'n', 'e', 'd', ' ', 'M',
                                   'e', 's', 's', 'a', 'g', 'e', ':', '\n'};
#endif
```